### DISCUSSION TITLE
No module named 'biscuit.cli'

### DISCUSSION THREAD

**Discussion by mkh-user** [tomlin7/biscuit#461] [Q&A] [answered]
Hello, I was trying to install this editor but I encountered various errors.
I have previously installed Python 3.11, Git, pip, and so on.
I have tried these two methods:
1. from pip
Currently, when I use `pip install biscuit-editor`, everything is fine and it installs successfully, but when I use `biscuit` to run the program:
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Program Files\Python311\Scripts\biscuit.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'biscuit.cli'
```
2. from releses
I downloaded and ran the latest version, but it encounters these errors and closes:
![image](https://github.com/user-attachments/assets/2c5d1d73-e2fd-436e-8e79-48b51d744fc5)

(All in Windows10 X64)

**Comment by tomlin7** [comment, marked as answer]
@mkh-user thank you for opening this discussion, but let's move this to an issue.
